system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -1677,7 +1669,6 @@ LDAPSECURITY_API IAuthMap *newDefaultAuthMap(IPropertyTree* config)
             loc_iter->next();
         }
         loc_iter->Release();
```

Comment:
Seems this line should also be removed since the Owned will do the release in during destruction. Unless there is something else going on elsewhere that is not immediately apparent. Removal is consistent with the other change in the PR.